### ISSUE TITLE
Data processing

### ISSUE THREAD

**Issue by Alexi2Cat** [MyParadise21/Mamba-SEUNet#3]
Hello, voicebank + demand is divided into two splits (test, train) , but the training also asks for a validation split. How has the split been done to replicate the paper results? Thanks

**Comment by MichaelChen147** [comment]
maybe validation split is the same as test split, such as p232 and p257.

**Comment by anasvaf** [comment]
Hello. Do you have any updates on the validation split? :)

--------------------------------------------------------------
Just realized that it is indeed the same as the test split as @MichaelChen147 mentioned.
See here: https://github.com/RoyChao19477/SEMamba/blob/main/data/valid_clean.json

**Comment by Alexi2Cat** [comment]
If the model is trained regarding validation loss, and the test and validation set are the same, there is a data leaking issue, isn't it?

**Comment by anasvaf** [comment]
@Alexi2Cat indeed. Maybe a manual split of the train set into train/validation would be better to evaluate the power of this network on the test set of voicebank+demand

**Comment by Alexi2Cat** [comment]
@anasvaf A few days ago I threw a training using a manual split of the test set as validation, to keep training data as it is. It is still running, we'll see how it performs on the test split when finished.
